tools/targets/__init__.py

```diff
@@ -130,8 +130,15 @@ class Target(namedtuple("Target", "name json_data resolution_order resolution_or
     @cached
     def get_json_target_data():
         """Load the description of JSON target data"""
-        return json_file_to_dict(Target.__targets_json_location or
-                                 Target.__targets_json_location_default)
+        targets = json_file_to_dict(Target.__targets_json_location or
+                                     Target.__targets_json_location_default)
+
+        # If extra_targets.json exists in working directory load it over the top
+        extra = os.path.join('.', 'extra_targets.json')
+        if os.path.exists(extra):
+            targets.update(json_file_to_dict(extra))
```

Comment:
Yeah that was my intention, so that you can replace/change/fix existing ones if desired. 

Obviously if this is likely to confuse people it could be merged in a more intelligent way (I've got a good recursive merge function if needed from another project)